component/Rating/rating.css

```diff
@@ -0,0 +1,27 @@
+.rating_container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+.rating_text {
+    font-size: 2rem;
+}
+
+i.fa-star {
```

Comment:
Give a class name like rating-star instead of i.star
Because you might have star icon in other places which might get the same style

Reply:
ok, I will change.